concepts/profile/realworldusage.md

```diff
@@ -0,0 +1,135 @@
+
+![](images/shared-computer.png)
+
+In this case, every user who logged into this one computer with a different OS user profile would show up as a different client ID. We have no way of knowing they came from the same computer. 
+
+Furthermore, if the computer wiped the user directory after use, like Guest accounts and university computer labs often do, then they would show up as a **new** client ID every time they logged in, even if they have used the same computer multiple times. This use case could inflate new profile counts. 
+
+Similar to Multi-Profile Users, in this use case, there would be only one download event and install event (assuming normal funnel and stub installer), but multiple client ID's. 
+
+#### Cloned Machines
+
+In this case, there are actually multiple users with computers that all share the same disk image at some point. 
+
+Think of the situation where the IT staff sets up the computer for a new hire at a company. Instead of going through to trouble of installing all the required programs and setting them up correctly for each computer, they'll do it once on one computer, save the disk image, and simply copy it over each time they need to issue a new machine. 
+
+Or think of the case where the IT staff of a library needs to set up 2 dozen machines at once. 
+
+![](images/cloned-machines.png)
+
+In this case, depending on the state of the disk image when it was copied, we could see multiple client ID's for each user+machine, or we could see all the user+machines sharing the same client ID. 
+
+If the disk image was copied after a Firefox profile was created, then the old user+machine and new user+machine will share the same client ID, and be submitting pings to us concurrently. 
+
+If the disk image was copied after the Firefox installation but before an initial Firefox profile was created, then each user+machine will get their own Firefox profile and client ID when they run Firefox for the first time. 
+
+As with the Multi-Profile User and Shared Computer case, even though there could be multiple Firefox profiles in this use case, there will only be one download and install event.  
+
+#### Migrations
+
+##### Type 1: Migrate Disk Image
+
+A user has a backup of their disk image and when they switch to a new computer or their current computer crashes, they simply reboot from the old disk image. 
+
+![](images/migration-1.png)
+
+In this case, the old machine and the new machine will just share the same client ID (assuming that the disk was copied after a Firefox profile was created). In fact, it will look exactly like the Cloned Machines case, except that instead of sending pings concurrently, they'll be sending us pings first from the old machine and then from the new machine. 
+
+Also, it should be noted that their Firefox profile will 'revert' back to the state that it was in when the disk image was copied, essentially starting over from the past. 
```

Comment:
(and any unsent pings in the image will be resent)

Reply:
good point, will add